### ISSUE TITLE
Stream Admin

### ISSUE THREAD

**Issue by thekeele** [thekeele/sentweet#12]
Auth has been added to sentweet for routes on the `/admin` scope.

We'd like an Admin page where we can manage the running streams in the system.

#### `GET /admin/streams`
List all running streams with whatever information is available. Call `Bitfeels.all_streams()` and display the results.

#### `POST /admin/streams`
Create a new twitter stream with the following fields:
`user_name`
`twitter_track`
`twitter_filter_level`
Call `Bitfeels.twitter_stream/3`

The `user_name` should map to the name that put into the Auth dialog box. This information _should_ be stored in the connection, i.e. `Plug.Conn`.

#### `DELETE /admin/streams/:id`
Delete a single running stream. Call `Bitfeels.stop_stream/2` with `user_name` and `twitter_track`

This is typically CRUD application stuff and should be able to find examples of best practices online.

I suggest breaking this into 3 PRs, one for each bit of functionality.

prerequisite for #13

**Comment by rcehemann** [comment]
The instructions are pretty clear to me.. not 100% sure on what all needs to be added in order to render a new page but I can figure that out. However, I'm having some trouble getting off the ground after the merge of the last PR.

`mix phx.server`

```
[info] Running SenTweetWeb.Endpoint with cowboy 2.7.0 at 0.0.0.0:4000 (http)
[info] Access SenTweetWeb.Endpoint at http://localhost:4000
[info] Application sen_tweet exited: SenTweet.Application.start(:normal, []) returned an error: shutdown: failed to start child: SenTweet.Bitfeels
    ** (EXIT) an exception was raised:
        ** (MatchError) no match of right hand side value: {:error, {:error, {:options, {:sslv3, {:versions, [:"tlsv1.2", :"tlsv1.1", :tlsv1, :sslv3]}}}}}
            (sen_tweet) lib/sen_tweet/bitfeels.ex:11: SenTweet.Bitfeels.init/1
            (stdlib) gen_server.erl:417: :gen_server.init_it/2
            (stdlib) gen_server.erl:385: :gen_server.init_it/6
            (stdlib) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
** (Mix) Could not start application sen_tweet: SenTweet.Application.start(:normal, []) returned an error: shutdown: failed to start child: SenTweet.Bitfeels
    ** (EXIT) an exception was raised:
        ** (MatchError) no match of right hand side value: {:error, {:error, {:options, {:sslv3, {:versions, [:"tlsv1.2", :"tlsv1.1", :tlsv1, :sslv3]}}}}}
            (sen_tweet) lib/sen_tweet/bitfeels.ex:11: SenTweet.Bitfeels.init/1
            (stdlib) gen_server.erl:417: :gen_server.init_it/2
            (stdlib) gen_server.erl:385: :gen_server.init_it/6
            (stdlib) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
```

I have done `mix deps.clean --all; mix deps.get` to no avail.

Any idea what's going on?

**Comment by thekeele** [comment]
Oh my, this is #25 

I thought this was only a production bug. What are your elixir / erlang versions?

**Comment by thekeele** [comment]
Development is still working for me.

Trying blowing away `_build` and `deps`
```
$ rm -rf _build
$ rm -rf deps
```

**Comment by rcehemann** [comment]
I'm running Erlang/OTP 23 and Elixir 1.9.1, but it seems elixir was built with Erlang/OTP 22. I'll get everything up to the latest and see if that makes a difference. No luck `rm -rf`'ing build and deps.

**Comment by rcehemann** [comment]
Tested on Elixir 1.10.1 compiled with Erlang/OTP 23 - same error

**Comment by thekeele** [comment]
Hmm might be OTP 23. I believe the server is running OTP 23.

My dev versions
```
Erlang/OTP 22 [erts-10.5] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe]

Interactive Elixir (1.10.0) - press Ctrl+C to exit (type h() ENTER for help)
```

**Comment by thekeele** [comment]
Fixed this bug on production by going back to OTP 22.

**Comment by rcehemann** [comment]
:fire: :fire: fixed for me with OTP 22 / Ex 1.10.3. now I can get down to bidness

**Comment by rcehemann** [comment]
Looks like you have your opportunity to dig down into the erlang innards and learn some stuff ^^
